docs/troubleshooting/index.mdx

```diff
@@ -35,6 +37,61 @@ Alpine is missing some dependencies that [are required](https://github.com/datre
 Adding the following command to the CI should fix this: `apk add curl bash`
 This will add curl and bash commands which our install scripts require in order to run.
 
+---
+
+## Schema validation failures
+
+If you get the following message in your CLI: 
+```terminal
+❌  k8s schema validation error: could not find schema for [KIND]  
+``` 
+It means that the resource's schema which is used to perform the schema valiation was not found, and it can be caused by any of the following-
+
+### Wrong K8s version is used in the schema validation check
```

Comment:
I think you go too much into details here which make it hard to read and understand

Reply:
To validate the schema, the CLI uses the default schema version configured in your account, or the value of the `--schema-version` flag.  
To fix this failure, change the schema version as shown [here](https://hub.datree.io/setup/schema-validation#changing-the-default-schema-version).